pkg/network/render.go

```diff
@@ -409,3 +419,22 @@ func renderMultusAdmissionController(conf *operv1.NetworkSpec, manifestDir strin
 	out = append(out, objs...)
 	return out, nil
 }
+
+// renderDiagnostics renders the connectivity checks
+func renderNetworkDiagnostics(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Unstructured, error) {
```

Comment:
nit: the comment and the function name are different

Reply:
done